modules/mono/editor/bindings_generator.cpp

```diff
@@ -3072,6 +3079,9 @@ bool BindingsGenerator::_arg_default_value_from_variant(const Variant &p_val, Ar
 			r_iarg.default_argument = "null";
 			break;
 		case Variant::ARRAY:
+			r_iarg.default_argument = "new %s {}";
```

Comment:
```suggestion
			r_iarg.default_argument = "new %s { }";
```
So that formatting the generated code matches the code we generate.

Reply:
Fixed.